katas/content/multi_qubit_gates/amplitudes_swap/solution.md

```diff
@@ -0,0 +1,58 @@
+### Solution 1 - using intrinsic gates
```

Comment:
```suggestion
```

We don't use solution headers now. You can refer to `solution.md` for `arbitary_controls` in this `multi_qubit_gates` kata.